app/routes/circulars/circulars.lib.ts

```diff
@@ -134,13 +134,17 @@ export function formatIsValid(format: string): format is CircularFormat {
 
 /** For updated dates, check that the date is valid */
 export function dateIsValid(date?: string, time?: string) {
-  return !Number.isNaN(Date.parse(`${date}T${time}:00.000Z`))
+  return !Number.isNaN(Date.parse(`${date}T${time}.000Z`))
 }
 
 export function submitterIsValid(submitter?: string) {
   return Boolean(submitter)
 }
 
+export function datetimeIsValid(date: string) {
```

Comment:
Rename for consistent case:

```suggestion
export function dateTimeIsValid(date: string) {
```